clients/drcachesim/scheduler/scheduler.h

```diff
@@ -778,6 +778,18 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
         return &outputs_[ordinal].stream;
     }
 
+    /**
+     * Returns the cpu number for the 'ordinal'-th output stream. This is only valid for
+     * MAP_TO_RECORDED_OUTPUT and returns -1 for other mappings.
+     */
+    virtual int
+    get_output_cpuid(output_ordinal_t ordinal)
```

Comment:
We figured this would be on the stream, not the scheduler.  We may also want this for other modes, including core-oriented analysis tool modes #5694, where it has to be on the stream as the scheduler is not exposed through the analyzer framework.

Reply:
I ended up needing this for #5694 and I put it into PR #6279 -- does that meet your needs as well?

Reply:
Thanks! Looks like this should be good.